src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Collections.Specialized;
+using System.ComponentModel;
```

Comment:
what is this used for?

Reply:
We're using PropertyChangedEventArgs when subscribing to the NodeModel.State change